docs/source/ecf.rst

```diff
@@ -18,6 +18,12 @@ suffix
 ''''''
 Data file suffix (e.g. uncal).
 
+
+pmap
```

Comment:
In the Conversation tab, you mentioned explicitly calling `jwst` vs `hst`.  Is there a chance users will want to use the `pmap` variable to reference an HST context?  Or maybe a future telescope, like Pandora?  If so, the parameter could be `jwst_1089`.  If not, then the current implementation is fine.

Reply:
So I'm currently working on tweaks to automatically add the instrument to that pmap name. Generally speaking this works well, but in the case where you work on two observatories in the same python instance then there appears to be some kind of bug with the crds package which doesn't allow you to switch (it warns me that I'm using hst for jwst data); this would be fairly rare in the real world, but happens every time we run the pytest suite... I've made some progress on this, but just need to figure out one more tweak

Reply:
Since it isn't just a matter of passing in a different pmap and we need to do some more annoying updates to crds variables, I think it's worth just leaving it in the way it's currently setup